catalog/dags/popularity/popularity_refresh_types.py

```diff
@@ -57,5 +67,9 @@ def __post_init__(self):
         media_type="image",
         refresh_metrics_timeout=timedelta(hours=24),
     ),
-    PopularityRefresh(media_type="audio"),
+    PopularityRefresh(
+        media_type="audio",
+        # Poke every minute, instead of every thirty minutes
+        poke_interval=int(os.getenv("DATA_REFRESH_POKE_INTERVAL", 60)),
```

Comment:
Is there a better way to manage this default? 🤔 It feels weird running the exact same steps as we do for building `REFRESH_POKE_INTERVAL`, but with a different default 😅

Reply:
Agreed! The problem is that if you set a poke interval via the environment variable, I want it to override all other configurations. So when you set a `poke_interval` in the PopularityRefresh configuration, it _should_ override the normal default, but _not_ the env var. 

REFRESH_POKE_INTERVAL is used in several other places including the DataRefresh configurations. We could have separate environment variables for the, but that seems unnecessary, or we could make them inherit from a common configuration which handles the defaults in one place?

Reply:
That all makes sense, we can keep it as-is!